ThemeDecorator/ThemeDecorator.js

```diff
@@ -177,7 +177,7 @@ const ThemeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 		[css.bg]: !overlay
 	});
 
-	let spotlightInputType = {};
+	const spotlightInputType = {request: null};
```

Comment:
How about renaming this as below?
```suggestion
	const requestInputType = null;
```